### DISCUSSION TITLE
Android CPU Usage (due to software decoding?)

### DISCUSSION THREAD

**Discussion by larryqiann** [rustdesk/rustdesk#4415] [Q&A]
Hi all,

I am attempting to use latest Nightlies on Android phone to connect to remote machines on both macOS and Windows. THe H264 and H265 options aren't available. While the functionality is working as expected, it appears that especially at a high resolution (4K), the CPU usage of the phone is extremely high. The CPU temperature becomes extremely hot, and after a few seconds, the app crashes. It seems to function fine at 1080p, but it is quite choppy compared to connecting from a more powerful PC. 

The processor in the phone is Snapdragon 888 and I have tried with 8 Gen 2 which both should have hardware support for decoding video. Is it possible to enable this functionality so that the phone can be used without heavy CPU load?

Any assistance would be greatly appreciated!

**Comment by rustdesk** [comment]
in plan, but not high priority.
